drivers/shared/executor/utils.go

```diff
@@ -75,10 +80,14 @@ func CreateExecutor(logger hclog.Logger, driverConfig *base.ClientDriverConfig,
 
 // CreateExecutorWithConfig launches a plugin with a given plugin config
 func CreateExecutorWithConfig(reattachConfig *plugin.ReattachConfig, logger hclog.Logger) (Executor, *plugin.Client, error) {
+	p := &ExecutorPlugin{
+		logger: logger,
+	}
```

Comment:
When re-attaching, the FSIsolation field isn't set - it seems unnecessary, but not sure if causes confusion.